<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LyricsX</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="LrcEditor" name="lrcEditor"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>541</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="action_Reload_With_Encoding"/>
    <addaction name="separator"/>
    <addaction name="action_Save"/>
    <addaction name="action_Save_as"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="action_Preferences"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <widget class="QMenu" name="menuLyrics_Rearrange">
     <property name="title">
      <string>Lyrics Rearrange</string>
     </property>
     <addaction name="action_Merge_Lyrics"/>
     <addaction name="action_Expand_Marks"/>
    </widget>
    <addaction name="action_Undo"/>
    <addaction name="action_Redo"/>
    <addaction name="separator"/>
    <addaction name="action_Insert_mark"/>
    <addaction name="action_Delete_mark"/>
    <addaction name="action_Remove_all_marks"/>
    <addaction name="separator"/>
    <addaction name="action_Add_music_info"/>
    <addaction name="separator"/>
    <addaction name="menuLyrics_Rearrange"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Preferences">
   <property name="text">
    <string>&amp;Preferences...</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="icon">
    <iconset theme="document-open">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset theme="document-save">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Save_as">
   <property name="icon">
    <iconset theme="document-save-as">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="action_Undo">
   <property name="icon">
    <iconset theme="edit-undo">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_Redo">
   <property name="icon">
    <iconset theme="edit-redo">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="action_Insert_mark">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/insert-mark.png</normaloff>:/icons/insert-mark.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Insert mark</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="action_Delete_mark">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/remove-mark.png</normaloff>:/icons/remove-mark.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Delete mark</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_Add_music_info">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/apply-music-info.png</normaloff>:/icons/apply-music-info.png</iconset>
   </property>
   <property name="text">
    <string>Add music info</string>
   </property>
  </action>
  <action name="action_Remove_all_marks">
   <property name="text">
    <string>Remove all marks</string>
   </property>
  </action>
  <action name="action_Merge_Lyrics">
   <property name="text">
    <string>Merge Lyrics</string>
   </property>
  </action>
  <action name="action_Expand_Marks">
   <property name="text">
    <string>Expand Marks</string>
   </property>
  </action>
  <action name="action_Reload_With_Encoding">
   <property name="text">
    <string>Reload With Encoding...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LrcEditor</class>
   <extends>QTextEdit</extends>
   <header location="global">lrceditor.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
